src/ansi-c/expr2c.cpp

```diff
@@ -860,6 +840,42 @@ std::string expr2ct::convert_array_type(
     src.subtype(), qualifiers, declarator_str+array_suffix);
 }
 
+std::string expr2ct::convert_pointer_type(
+  const typet &src,
+  const c_qualifierst &qualifiers,
+  const std::string &declarator)
+{
+    c_qualifierst new_qualifiers(qualifiers);
+    new_qualifiers.read(src);
+    std::string q=new_qualifiers.as_string();
+
+    c_qualifierst sub_qualifiers;
```

Comment:
Is this the right set of qualifiers if the modified behaviour is activated (for example, ensure we have tests for const pointers to const arrays, const pointers to non const arrays etc.)